### ISSUE TITLE
Try to add gradients function of keras in plaidml-v1 branch

### ISSUE THREAD

**Issue by terrs** [plaidml/plaidml#1709]
Hi guys, 

    When I ran plaidbench with plaidml-v1,  the error message ''The Keras backend function graidents is not yet implemented' was reported, so I added some codes.
   I wonder if this is the right way，hope to get advice，

branch: 
plaidml-v1  commit 8f9f6aacb7480bff5285d53b1357481236d3bcaf

OS: 
macOS Catalina 10.15.7

command line:
#plaidbench --print-stacktraces --batch-size 16 keras --train mobilenet

[diff_add_kerasgradient.txt](https://github.com/plaidml/plaidml/files/6036409/diff_add_kerasgradient.txt)
